### PR TITLE
Add remote write to Cortex

### PR DESCRIPTION
**Pull request by omegas27**
Towards https://github.com/giantswarm/giantswarm/issues/13092

### CODE REVIEW

**Comment by omegas27** [comment]
image
Tested on Gauss

File: helm/prometheus-meta-operator/templates/secret.yaml [hunk 1]
```diff
@@ -8,5 +8,10 @@ metadata:
 stringData:
   secret.yml: |
     service:
+      prometheus:
+        remoteWrite:
+          basicAuth:
+            username: 10755
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/secret.yaml:14]: Can we also have the username templated ? Nice to have, I am ok if we can't.

**Review by TheoBrigitte — APPROVED** [review]
LGTM when tested

**Comment by TheoBrigitte** [comment]
> image
> Tested on Gauss

Why do we loose labels like `customer`, `installation`, `pipeline`, `provider`, and `region`? I think we do need those labels.

File: helm/prometheus-meta-operator/templates/secret.yaml (hunk 1 above)
Comment (omegas27) [reply helm/prometheus-meta-operator/templates/secret.yaml:14]: Right now, not really, we can add that in the helm values but it is not in the installations repo. You can add a PR for it though :)

**Comment by omegas27** [comment]
We are loosing some labels because what is used is the external labels witch we did not really fill per prometheus

File: helm/prometheus-meta-operator/templates/secret.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/prometheus-meta-operator/templates/secret.yaml:14]: Ok fine

**Review by TheoBrigitte — APPROVED** [review]

**Comment by omegas27** [comment]
image
tested multiple times on Gauss :D

**Pull request merged**